### DISCUSSION TITLE
Population distribution (demography) layer naming conventions

### DISCUSSION THREAD

**Discussion by mireslav** [INSPIRE-MIF/helpdesk#116] [Q&A]
The statistics community discussed about layer naming conventions for the Population distribution (demography) PD theme

Statistics Sweden was trying to configure INSPIRE compliant view services for population distribution (population in 1 x 1 km grids) and was struggeling to understand the naming conventions to be used for the layers in the service. When validating view services, the ETF validator complained about the name element of the layers in the WMS because according to the validator the names do not follow specs. They could not find any guidance in the TG or in the INSPIRE layer register under PD. Hence, the services do not follow the specs but there seems to be no specs to follow.

The lack of a layer name for PD is the result of the PD theme not carrying geometry itself but deriving it from the SU (statistical units) theme. Therefore a valid geographical reference for PD data would be a SU.VectorStatisticalUnit or SU.GridStatisticalUnit layer. The human health theme (somewhat similar to PD) has a HH.HealthStatisticalData layer which is of HealthStatisticalUnit type, so the analogy can be carried to the SU-PD tandem.

The Netherlands created their own naming convention:
pd-nl-nuts2-2018
pd-nl-lau-2018
pd-nl-grid-2012 (metadata in English can be found here: https://www.nationaalgeoregister.nl/geonetwork/srv/dut/catalog.search#/metadata/76541ebb-a554-4540-a017-399d4bb4a860)
as well as for HH:
hh-nl-nuts2-doctors-2017
hh-nl-nuts2-death-2016
Metadata link: https://www.nationaalgeoregister.nl/geonetwork/srv/dut/catalog.search#/metadata/12bb57cd-73e0-41ba-a1e9-6d986b9109d3

Austria also uses their own layer naming, imitating the naming of the other themes: PD.StatisticalDistribution. And for the health theme they use the HH.HealthStatisticalData layer.

Any other suggestions as to layer naming for PD?

**Comment by sMorrone** [comment]
Dear @mireslav 
not easy to agree on layer naming conventions! I add the "community input wanted" label, though, and leave this issue still open so that maybe other examples can come.
In the meantime, if nothing changed regarding the ETF complains, a new issue could be opened in the validator helpdesk

**Comment by sMorrone** [comment]
issue is linked to https://github.com/INSPIRE-MIF/helpdesk-validator/issues/39
